exercises/armstrong-numbers/.meta/solutions/armstrong_numbers.rb

```diff
@@ -0,0 +1,10 @@
+class ArmstrongNumbers
+  def self.is_valid?(number)
```

Comment:
I like `self.valid?` better than `self.armstrong?`, although it's not really about validity, is it? I can't think of a better name though. 🤔

Reply:
In response to this comment I heavily considered suggesting `is?` because I thought it would be slightly amusing... but `ArmstrongNumbers.is?(10)` actually works in the opposite direction compared to `is_a?` so it would be unconventional.

If using `include?`, then `ArmstrongNumbers.include?(10)` *does* read naturally, but I think it would be surprising because it hints that you can use other Enumerable methods on ArmstrongNumbers (because `include?` is on Enumerable) but you cannot.

Reply:
I really like `include?` :+1:

Reply:
Including `Enumerable` is not really relevant to this PR but is something that could be interesting to explore as part of a solution.